algoliasearch/answers_client.py

```diff
@@ -0,0 +1,59 @@
+from typing import Optional, Union, Dict, Any
+
+from algoliasearch.configs import AnswersConfig
+from algoliasearch.helpers import is_async_available
+from algoliasearch.http.request_options import RequestOptions
+from algoliasearch.http.requester import Requester
+from algoliasearch.http.transporter import Transporter
+from algoliasearch.http.verb import Verb
+
+
+class AnswersClient(object):
+    def __init__(self, transporter, config):
+        # type: (Transporter, AnswersConfig) -> None
+
+        self._transporter = transporter
+        self._config = config
+
+    @staticmethod
+    def create(app_id=None, api_key=None):
+        # type: (Optional[str], Optional[str]) -> AnswersClient  # noqa: E501
+
+        config = AnswersConfig(app_id, api_key)
+
+        return AnswersClient.create_with_config(config)
+
+    @staticmethod
+    def create_with_config(config):
+        # type: (AnswersConfig) -> AnswersClient
+
+        requester = Requester()
+        transporter = Transporter(requester, config)
+
+        client = AnswersClient(transporter, config)
+
+        if is_async_available():
+            from algoliasearch.answers_client_async import (
+                AnswersClientAsync,
+            )
```

Comment:
Since you are not importing more class, you should do this
```suggestion
            from algoliasearch.answers_client_async import AnswersClientAsync
```